.github/workflows/build.yml

```diff
@@ -0,0 +1,326 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
```

Comment:
Make it trigger on demand instead of cron due to concern with vhdStorageAccount cleanup

Reply:
Done